### PR TITLE
1.10.0 Doc Updates

### PR DESCRIPTION
**Pull request by oeggert**
This PR will merge these doc updates:

- #1793 
- #1789 
- #1787 
- #1780 
- #1776 
- #1799

1.10.0 planned release is March 14, 2023.

### CODE REVIEW

**Comment by mDuo13** [comment]
The broken links are not caused by this PR; see #1808 for the fix.

File: content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.md [hunk 1]
```diff
@@ -67,18 +67,22 @@ There are several options which can be either enabled or disabled for an account
...
+| `lsfAMM` :not_enabled:            | `0x02000000` | 33554432          | (None)                            | This account is an Automated Market Maker instance. :not_enabled: |
+| `lsfDefaultRipple`                | `0x00800000` | 8388608           | `asfDefaultRipple`                | Enable [rippling](rippling.html) on this addresses's trust lines by default. Required for issuing addresses; discouraged for others. |
+| `lsfDepositAuth`                  | `0x01000000` | 16777216          | `asfDepositAuth`                  | This account can only receive funds from transactions it sends, and from [preauthorized](depositauth.html#preauthorization) accounts. (It has [DepositAuth](depositauth.html) enabled.) |
+| `lsfDisableMaster`                | `0x00100000` | 1048576           | `asfDisableMaster`                | Disallows use of the master key to sign transactions for this account. |
+| `lsfDisallowIncomingCheck`        | `0x08000000` | 134217728         | `asfDisallowIncomingCheck`        | This account is blocking incoming checks. _(Added by the [DisallowIncoming amendment][].)_ |
```
Comment (DennisDawson) [inline content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.md]: Curious about the present participle form. Perhaps "This account blocks..." works just as well? Same applies to lines 77-79.

File: content/references/protocol-reference/transactions/transaction-types/nftokencreateoffer.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ Besides errors that can occur for all transactions, {{currentpage.name}} transac
 | `tecNO_ENTRY`                    | The `NFToken` is not owned by the expected account. |
 | `tecNO_ISSUER`                   | The issuer specified in the `Amount` field does not exist. |
 | `tecNO_LINE`                     | The `Amount` field is denominated in fungible tokens, but the `NFToken`'s issuer does not have a trust line for those tokens and the `NFToken` does not have the [`lsfTrustLine` flag](nftoken.html#nftoken-flags) enabled. |
+| `tecNO_PERMISSION`               | The `Destination` is blocking incoming NFToken offers. _(Added by the [DisallowIncoming amendment][].)_
```
Comment (DennisDawson) [inline content/references/protocol-reference/transactions/transaction-types/nftokencreateoffer.md]: Suggest "The `Destination` blocks...."

**Review by DennisDawson — APPROVED** [review]
Consider changing the present participle to present tense (_is blocking_ to _blocks_), I don't see anything else.

**Pull request merged**